src/platform/forms-system/src/js/review/StringField.jsx

```diff
@@ -13,18 +14,67 @@ import {
  * in the config
  */
 export default function StringField(props) {
-  const { registry, schema, uiSchema, formData } = props;
-  const uiOptions = getUiOptions(uiSchema);
+  const {
+    registry = getDefaultRegistry(),
+    schema,
+    uiSchema,
+    formData,
+    children,
+    formContext,
+    id,
+  } = props;
+  const realUiSchema = getNestedUISchema(uiSchema);
```

Comment:
I wonder if it would be better to have a HOC that runs `getNestedUISchema` and passes data into `StringField` so that we do not have to update this component?

Reply:
Do we need `getNestedUISchema` if we do [something like this](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/definitions/email.js) instead?